demo/package.json

```diff
@@ -33,8 +33,7 @@
     "mocha": "~3.5.0",
     "nativescript-css-loader": "~0.26.0",
     "nativescript-dev-appium": "^3.1.0",
-    "nativescript-dev-typescript": "^0.6.0",
-    "nativescript-dev-webpack": "^0.9.0",
+    "nativescript-dev-webpack": "^0.10.1",
```

Comment:
I noticed nativescript-dev-typescript is removed but I'm not sure if it's intentionally so point it out here.

Reply:
nice catch